src/apollo/queries.js

```diff
@@ -19,6 +19,7 @@ export const GET_RUN_METADATA = gql`
     runMetadata(runIds: $runIds) {
       id
       author
+      bookmark
```

Comment:
not in scope of your PR but this query has duplicated `id` field. Can we quickly remove it here?

Reply:
Absolutely. It's gone.